pandas/core/arrays/masked.py

```diff
@@ -80,6 +80,8 @@ class BaseMaskedArray(OpsMixin, ExtensionArray):
 
     # The value used to fill '_data' to avoid upcasting
     _internal_fill_value: Scalar
+    _data: np.ndarray
+    _mask: np.ndarray[Any, bool]
```

Comment:
np.ndarray in numpy 1.20 is not generic so although mypy is happy with type parameters on Any, this will raise errors when we transition to numpy types.

Reply:
removed the `[Any, bool]` part of this.  is there an approximate calendar for the transition to numpy types?

Reply:
I think these are in 1.20 (releaseing blocking on arrow update atm)

Reply:
> is there an approximate calendar for the transition to numpy types?

I updated to 1.20.0rc2 locally and there were no changes to the mypy errors. ( I've not yet checked the commit history to see if there were any changes)

Once 1.20 is released, and we pick it up on CI we will see the errors in #36092. (I'll merge master and make a start on updating now) 

I assume that we will pin numpy in ci while we discuss how to sort out the errors. (once we know what the status is with the released numpy)